content/_index.md

```diff
@@ -24,24 +24,35 @@ section = "blog"
 url = "/blog/"
 weight = 20
 
+[[extra.menu.main]]
+name = "Events"
+section = "events"
+url = "/events/"
+weight = 20
+
 [[extra.list]]
 title = "Rust Resources"
 content = 'Get all the resources for you to get started from basics to advanced topics'
+url = "#"
 
 [[extra.list]]
 title = "Sub Communities"
 content = 'Information and PoC of communities near you to take part in and geek out on Rust together'
+url = "/docs/sub-communities/details"
```

Comment:
thanks for adding this! can you also point to Events at "https://rustindia.github.io/docs/sub-communities/details/". 

continue the paragrph with ". Do checkout the latest [events conducted by these communities.](/events)"

Reply:
No problem, Yes.